vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -206,46 +206,77 @@
 
       <Function Name="isroot" IsBound="true">
         <Annotation Term="Core.Description"
-          String="Returns true, if and only if the value of the node property of the specified hierarchy is the root of the hierarchy" />
+          String="Returns true for a given entity, if and only if the value of the node property of the specified hierarchy annotation is the root of the hierarchy in the given entity set of hierarchy nodes" />
         <Parameter Name="Entity" Type="Edm.EntityType" Nullable="false" />
-        <Parameter Name="Hierarchy" Type="Edm.String" Nullable="false" />
+        <Parameter Name="Nodes" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Name of the entity set holding the nodes of the recursive hierarchy" />

```

Comment:
```suggestion
          <Annotation Term="Core.Description" String="Name of the entity set to which the `RecursiveHierarchy` annotation is applied and that contains the nodes of the specified hierarchy" />
```

Reply:
Well, actually the term is applied, and it is applied to the entity type of the entity set.

"Name of the entity set containing the nodes of the hierarchy specified by the `RecursiveHierarchy` annotation on its entity type"

Reply:
Why is this parameter needed at all? Can't we simply assume this is the entity set of the binding parameter?